packages/ripple-keypairs/src/index.ts

```diff
@@ -1,33 +1,41 @@
-import brorand = require('brorand')
-import * as hashjs from 'hash.js'
-import * as elliptic from 'elliptic'
-
 import * as addressCodec from 'ripple-address-codec'
-import { derivePrivateKey, accountPublicFromPublicGenerator } from './secp256k1'
-import * as utils from './utils'
 
-const Ed25519 = elliptic.eddsa('ed25519')
-const Secp256k1 = elliptic.ec('secp256k1')
+import { secp256k1 as nobleSecp256k1 } from '@noble/curves/secp256k1'
+import { ed25519 as nobleEd25519 } from '@noble/curves/ed25519'
+import { numberToBytesBE } from '@noble/curves/abstract/utils'
+import { ripemd160 } from '@xrplf/isomorphic/ripemd160'
```

Comment:
I thought the objective of using `@xrplf/isomorphic` was that you didn't have to use `@noble/...` directly

Reply:
Good point.  These were missed.

Reply:
@mvadari @ckniffen 

Wait, there's a difference between @noble/hashes and @noble/curves/* here, right?

We talked about whether to make an elliptic curve features wrapper/abstraction that could support a native implementation but decided to punt on it for now. (This PR is getting big enough)

Reply:
Also, note there's no "native implementation" library candidate as yet.

Reply:
Good call.  It was all blending together.  @mvadari using noble libs curves replaces elliptic as a dependency

Reply:
Basically you just get the tiny perf gains from using native hashing